MIGRATING.md

```diff
@@ -0,0 +1,140 @@
+# Migrating from qiskit-ibmq-provider>=0.16
+
+## Installation
+The Qiskit IBM Provider is now distributed as a separate PyPI package called `qiskit-ibm`. You can install the provider using pip:
+
+```bash
+pip install qiskit-ibm
+```
+
+## Breaking Changes
+1. The `IBMQ` global variable which was an instance of the `IBMQFactory` has been removed.
+2. `IBMQFactory` has been renamed to `IBMAccount` class.
+3. `AccountProvider` class has been renamed to `IBMProvider` class.
+
+    For example, if you are looking to quickly migrate your existing code:
+
+    Before
+    ```python
+    from qiskit import IBMQ
+    IBMQ.load_account()
+    provider = IBMQ.get_provider(hub='ibm-q', group='test', project='default')
+    simulator_backend = provider.get_backend('ibmq_qasm_simulator')
+    ```
+    After
+    ```python
+    from qiskit_ibm import IBMAccount
+    account = IBMAccount()
+    account.load_account() # or account.load()
```

Comment:
Yup, I can simplify that in one line.

Reply:
Where I was going with this is that by separating `IBMAccount()` and `loa_account()`, it appears that the new interface requires 2 lines to do what could be done with 1 line (`IBMQ.load_account()`) before. This is of course not true, since `IBMQ = IBMQFactory()` is just hidden.  But if the `After` is longer than `Before`, it gives the impression that the new interface is clunkier.

Reply:
Fixed!